runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -1565,8 +1565,8 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
          auto recv = client->getRecvData<TR_ResolvedJ9Method*, int32_t>();
          auto *owningMethod = std::get<0>(recv);
          int32_t cpIndex = std::get<1>(recv);
-         bool isUnresolvedInCP;
-         auto *handleMethod = static_cast<TR_ResolvedJ9Method *>(owningMethod->getResolvedHandleMethod(comp, cpIndex, &isUnresolvedInCP));
+         bool isUnresolvedInCP, isInvokeCacheAppendixNull;
```

Comment:
Here and in the `getResolvedDynamicMethod` case below, `isInvokeCacheAppendixNull` will remain uninitialized whenever the call is unresolved, and also (probably the most typical case) whenever the call is resolved but the appendix is non-null

This could be solved by initializing it to `false` in the declaration as you've done in Walker.cpp, but I would suggest (possibly in addition) that whenever the pointer is specified, `getResolvedHandleMethod()` and `getResolvedDynamicMethod()` should always set it. If we don't have a resolved null appendix, it can be set to false